tests/test_metadata.py

```diff
@@ -163,6 +163,9 @@ def _get_container_label_prefix(
         ]
         if LOCALHOST.system_info.arch == "x86_64"
         else []
+        + [
```

Comment:
No, this is the wrong place. What you just did is adding the distribution container on non-x86_64. Please add it after postgres

Reply:
Done.